backend/src/main/java/com/votogether/domain/auth/dto/request/AccessTokenRequest.java

```diff
@@ -0,0 +1,10 @@
+package com.votogether.domain.auth.dto.request;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "인증 토큰 재발급 요청")
+public record AccessTokenRequest(
```

Comment:
`P2`
```suggestion
public record RenewAccessTokenRequest(
```
재발급 의미를 조금 더 추가한 변수명은 어떠신가요 !

Reply:
`reissued`라는 prefix를 추가해보겠습니다!